spring-vault-core/src/main/java/org/springframework/vault/support/Certificate.java

```diff
@@ -161,4 +187,29 @@ static List<X509Certificate> getCertificates(String certificates) throws Certifi
 		return result;
 	}
 
+	/**
+	 * Retrieve the issuing CA certificates as list of {@link X509Certificate}.
+	 * @return the issuing CA {@link X509Certificate}.
+	 * @since 2.3.3
+	 */
+	public List<X509Certificate> getX509IssuerCertificates() {
```

Comment:
Note that this change is source-compatible, yet binary breaking and it will require users to recompile their code.

Reply:
@mp911de I thought this was allowed as as there is a matching signature and return type declared in a superclass now, but not sure. Should we introduce a different method?

Reply:
This was more of a fyi. For minor versions, that's okay.